install_files/ansible-base/group_vars/securedrop_application_server.yml

```diff
@@ -20,6 +20,17 @@ tor_instances:
   - service: journalist
     filename: app-journalist-aths
 
+tor_instances_v3:
+  - "{{ {'service': 'sshv3', 'filename': 'app-sshv3-aths'} if enable_ssh_over_tor else [] }}"
```

Comment:
The `else []` causes an empty list here—is that intentional? The other elements of `tor_instances_v3` are dicts, not lists. Haven't tested functionally yet to observe an error here, but it seems inconsistent.

Reply:
I copied that from the above lines in the same file https://github.com/freedomofpress/securedrop/blame/07fed0c56e89d7edb006a7aa9e0f850d3d516216/install_files/ansible-base/group_vars/securedrop_application_server.yml#L17  Seems to be added by @msheiny  2 years ago.